### PR TITLE
[TIMOB-1618] Android: Label property minimumFontSize not implemented

### PR DESCRIPTION
**Pull request by lgolu**
I've resolved the parity ticket for minimumFontSize property of the Label on Android (https://jira.appcelerator.org/browse/TIMOB-1618). 
If the property is set it scales the text to fit, same behaviour as on iPhone. It was tested with the Titanium app I'm working on at Atoss, cannot publish testing code.

### CODE REVIEW

File: android/titanium/src/java/org/appcelerator/titanium/TiC.java [hunk 1]
```diff
@@ -1208,7 +1208,12 @@
 	 * @module.api
 	 */
 	public static final String PROPERTY_FONT_SIZE = "font-size";
-
+	
+	/**
+	 * @module.api
+	 */
+	public static final String PROPERTY_MINIMUMFONTSIZE = "minimumFontSize";
```
Comment (collinprice) [inline android/titanium/src/java/org/appcelerator/titanium/TiC.java]: Might want to change `PROPERTY_MINIMUMFONTSIZE ` to `PROPERTY_MINIMUM_FONT_SIZE` to more align with Appcelerator's coding style.

**Comment by hansemannn** [comment]
@lgolu Can you address the comment above and update the repo to work with the latest master? Its a great addition to the SDK so we would appreciate it as part of the core. Thank you!

File: android/titanium/src/java/org/appcelerator/titanium/TiC.java (hunk 1 above)
Comment (lgolu) [inline android/titanium/src/java/org/appcelerator/titanium/TiC.java]: Done

**Comment by hansemannn** [comment]
@lgolu Thanks for addressing the comments! Can you please pull from master to resolve the merge conflicts? Then @fmerzadyan can check the CR and FT. Thanks!

**Comment by lgolu** [comment]
@hansemannn It would be great if you could resolve the merge. Thank you!

**Comment by hansemannn** [comment]
Thanks @lgolu! Fixed the merge conflicts, updated the indentation and the docs. Follow-up: https://github.com/appcelerator/titanium_mobile/pull/8724

**Pull request closed without merging**